socs/agents/acu/agent.py

```diff
@@ -236,15 +236,29 @@ def monitor(self, session, params):
         """
 
         session.set_status('running')
-        version = yield self.acu_read.http.Version()
-        self.log.info(version)
 
         # Note that session.data will get scanned, to assign data to
         # feed blocks.  Items in session.data that are themselves
         # dicts will parsed; but items (such as PlatformType and
         # StatusResponseRate) which are simple strings or floats will
         # be ignored for feed assignment.
-        session.data = {'PlatformType': self.acu_config['platform']}
+        session.data = {'PlatformType': self.acu_config['platform'],
+                        'connected': False}
+
+        last_complaint = 0
+        while True:
+            try:
+                version = yield self.acu_read.http.Version()
+                break
+            except Exception as e:
+                if time.time() - last_complaint > 3600:
```

Comment:
Is complaining every hour about a dropout often enough?

Reply:
I think so, because during that time the session.data will accurately reflect connected=False, and the monitor data won't be updating... no motion commands will be executable.  It will be pretty obvious that the monitoring is not working.